src/widget/hotcuedrag.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <QBuffer>
+#include <QMimeData>
+#include <QString>
+
+#include "track/cue.h"
+#include "track/trackid.h"
+
+const QString kDragDataType = QStringLiteral("hotcueDragInfo");
+
+class TrackId;
+
+struct HotcueDragInfo {
```

Comment:
should I move this and kDragDataType to mixxx::hotcuedrag, too?
and the implementation to cpp?

Reply:
Just did it.
and with `using namespace mixxx::hotcuedrag` it's all quite nice now.